### PR TITLE
11.0 mig account invoice line sequence

### PR DESCRIPTION
**Pull request by amcor**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Are you sure this one is still needed? AFAIK Odoo already includes a sequence and order by it.

File: account_invoice_line_sequence/__init__.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Copyright 2017 Camptocamp SA - Damien Crier, Alexandre Fayolle
```
Comment (nikul-serpentcs) [inline account_invoice_line_sequence/__init__.py]: @amcor Remove Copyright from all `__init__.py` file.

File: account_invoice_line_sequence/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+{
+    'name': 'Account Invoice line with sequence number',
+    'version': '11.0.1.0.0',
+    "summary": "Adds sequence field on invoice lines to manage its order.",
+    'category': 'Accounting',
+    'author': "Camptocamp, "
+              "Eficent, "
+              "Serpent CS, "
+              "Odoo Community Association (OCA)",
+    'website': 'http://www.camptocamp.com',
```
Comment (nikul-serpentcs) [inline account_invoice_line_sequence/__manifest__.py]: @amcor IMO Replace URL as per [this](https://agithub.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)

File: account_invoice_line_sequence/views/report_invoice.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_invoice_sequence_qweb" inherit_id="account.report_invoice_document">
+        <xpath expr="//table[@class='table table-condensed']/thead/tr/th[1]" position="before">
```
Comment (nikul-serpentcs) [inline account_invoice_line_sequence/views/report_invoice.xml]: @amcor IMO here use `hasclass`

File: account_invoice_line_sequence/views/report_invoice.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+            <th><strong>Sequence</strong></th>
+        </xpath>
+        <xpath expr="//table[@class='table table-condensed']/tbody/tr/td[1]" position="before">
```
Comment (nikul-serpentcs) [inline account_invoice_line_sequence/views/report_invoice.xml]: same here

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Comment by AaronHForgeFlow** [comment]
can you check travis?

**Comment by MiquelRForgeFlow** [comment]
please, squash last commits

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional tests good.

**Comment by AaronHForgeFlow** [comment]
> Are you sure this one is still needed? AFAIK Odoo already includes a sequence and order by it.

IMHO I think yes. The thing is, for big invoices with dozens of lines, it is really hard to now the line number, invoicing users spend to much time counting the lines. Also, when integrating this module with sale_order_line_sequence, it is very useful to now exactly we are invoicing the line number XX. In many use cases people just invoice by line number, not by product, and the line in the sale order must match the line in the invoice.

**Pull request merged**